ui/src/source.ts

```diff
@@ -118,18 +112,20 @@ enum Kind {
   FetchCommit = "FETCH_COMMIT",
   FetchCommits = "FETCH_COMMITS",
   FetchRevisions = "FETCH_REVISIONS",
-  Update = "UPDATE",
+  FetchObject = "FETCH_OBJECT",
 }
 
 interface FetchCommit extends event.Event<Kind> {
   kind: Kind.FetchCommit;
   projectId: string;
+  userId: string;
   sha1: string;
 }
 
 interface FetchCommits extends event.Event<Kind> {
   kind: Kind.FetchCommits;
   projectId: string;
+  userId: string;
```

Comment:
```suggestion
  userUrn: string;
```

Reply:
I don't think that's correct, it's the `PeerId` if I'm not mistaken. But not sure that's what we want to call it in the UI components..

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/620/commits/d5b10c171dddbf5b93c4a3b1493d25c1689799f0